redhat/ubi8-openjdk-11.yaml

```diff
@@ -3,7 +3,6 @@ osbs:
     container:
       compose:
         pulp_repos: true
-        inherit: true
```

Comment:
It's not clear what effect that will have for building the container against a pre-GA OpenJDK package. However, let's remove and add back should we need it in future.